src/rust/grapl-crypto/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+pub(crate) mod hasher;
+pub mod aead;
+pub mod asymmetric;
+pub mod rand_bytes;
+
+// Lets us serialize/ deserialize arrays > 32 bytes
```

Comment:
Nothing in this module seems to be concerned with arrays of greater than 32 bytes; am I missing something?